crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/staticdata/enums/AssessmentStatus.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.laa.crime.contribution.staticdata.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.Stream;
+
+@Getter
+@AllArgsConstructor
+public enum AssessmentStatus {
+    COMPLETE("COMPLETE"),
+    IN_PROGRESS("IN PROGRESS");
+
+    public final String value;
+
+    public static AssessmentStatus getFrom(String value) {
+        if (StringUtils.isBlank(value)) return null;
+
+        return Stream.of(AssessmentStatus.values())
+                .filter(assessmentStatus -> assessmentStatus.value.equals(value))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException(String.format("Assessment status with value: %s does not exist.", value)));
```

Comment:
can extract this out as a constant
i.e.
`private static final String EXCEPTION_MESSAGE_FORMAT = "Assessment status with value: %s does not exist.";
`